pom.xml

```diff
@@ -4,22 +4,19 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-
     <groupId>com.ryandw11</groupId>
     <artifactId>CustomStructures</artifactId>
     <version>1.8.2-DEV</version>
 
     <build>
         <sourceDirectory>src/main/java</sourceDirectory>
         <plugins>
-
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>16</source>
-                    <target>16</target>
+                    <release>17</release>
```

Comment:
Don't change the Java version of the plugin.

Reply:
I will revert it (I was just a bit of curious if there is a particular reason for using Java 16 instead of 17)